osu.Game.Rulesets.Mania/ManiaInputManager.cs

```diff
@@ -19,16 +19,8 @@ public ManiaInputManager(RulesetInfo ruleset, int variant)
 
     public enum ManiaAction
     {
-        [Description("Special 1")]
-        Special1 = 1,
-
-        [Description("Special 2")]
-        Special2,
-
-        // This offsets the start value of normal keys in-case we add more special keys
-        // above at a later time, without breaking replays/configs.
         [Description("Key 1")]
-        Key1 = 10,
+        Key1,
```

Comment:
If we wanted to make the enum match the key number, we could do this

```suggestion
        Key1 = 1,
```

Reply:
Applied this via 952a73b005e83eaea0167790462d78447eeba195 (smoogi agrees to it).

Reply:
Or not, something relies on zero index somewhere who knows.